src/components/infoBox/InfoBox.styled.ts

```diff
@@ -0,0 +1,40 @@
+import styled from "styled-components";
+
+import { themes } from "../../themes";
+
+export const InfoBoxStyledContainer = styled.div`
+	width: 75%;
+	font-size: 0.9rem;
+	background: rgba(155, 155, 155, 0.4);
+	box-shadow: 0 8px 32px 0 rgba(31, 38, 135, 0.37);
+	backdrop-filter: blur(5px);
+	-webkit-backdrop-filter: blur(5px);
+	border-radius: 10px;
+	border: 1px solid ${themes.light.background};
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+	align-items: center;
+	color: ${themes.light.background};
+	padding: 2px;
+	font-family: "Roboto", sans-serif;
```

Comment:
To roboto nie jest globalnym fontem, nie da się w jakimś globalnym pliku CSS ustalić rodziny fontów dla wszystkich/wybranych elementów?